module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,22 @@
+#include "vowel.hpp"
+#include <algorithm>
+#include <string>
+#include <vector>
+#include <iostream>
+void removeVowels(std::vector<std::string>& words)
+{
+    std::string vowels ("aeiouyAEIOUY");
+
+  for(auto& sentence : words)
+  {
+    auto it = std::remove_if(sentence.begin(), sentence.end(),[vowels](auto el){
+      if(find(vowels.begin(), vowels.end(), el) != vowels.end())
+      return true;
+      else return false;
```

Comment:
those returns look weird

Reply:
It will be much better with at least indentation, but braces are recommended also

Reply:
**if** statement checks **bool** value, so when you return **bool** value immediately after check you can **simplify** it and transform into single line instruction.

Reply:
Fixed